src/store/indexeddb-remote-backend.js

```diff
@@ -62,7 +64,7 @@ RemoteIndexedDBStoreBackend.prototype = {
      * @return {Promise} Resolved when the database is cleared.
      */
     clearDatabase: function() {
-        return this._doCmd('clearDatabase');
+        return this._startPromise.then(() => this._doCmd('clearDatabase'));
```

Comment:
No, because _setupWorker is only called once, from the constructor. We're just piggy-backing on the promise created there.